packages/swap/src/event-handlers/__tests__/swapEgressScheduled.test.ts

```diff
@@ -90,4 +103,71 @@ describe(swapEgressScheduled, () => {
       fees: [{ id: expect.any(BigInt), swapId: expect.any(BigInt) }],
     });
   });
+
+  it('updates an existing swap when the egress asset is not the native asset', async () => {
```

Comment:
Can we maybe check if the `getSwapRate` function was called in this test?

Reply:
the snapshot checks makes sure that the egress fee is three times the fee of the environment 🙂 but will add another assertion in the next pr